rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -47,3 +47,48 @@ spec:
             alert_team_handle: <!subteam^S03GF42RBE2>
             team: pipelines
             runbook_url: TBD
+        - alert: CorePipelineControllerRepeatedRestarts
+          expr: |
+            sum by (source_cluster) (increase(kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod=~"tekton-pipelines-controller-.*"}[2m])) > 0
+          for: 5m
+          labels:
+            severity: critical
+            slo: "true"
+          annotations:
+            summary: >-
+              Tekton controller is rapidly restarting.
+            description: >-
+              Tekton controller on cluster {{ $labels.source_cluster }} has restarted {{ $value }} times recently.
+            alert_team_handle: <!subteam^S03GF42RBE2>
+            team: pipelines
+            runbook_url: TBD
```

Comment:
I should also point out the importance of having a SOP for each SLO alert. It would be great to work on that as soon as possible :)

Reply:
Yes we'll be updating the SOPs i.e. https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/stonesoup/pipeline-service/slo-sli-document.yml?ref_type=heads as part of SRVKP-4523 and SRVKP-4524 as well as planning to add the app-sre panels in this PR per my earlier comments.

We'll also be updating https://gitlab.cee.redhat.com/konflux/docs/sop/-/tree/main/pipeline-service/slos?ref_type=heads adding new markdowns for the new alerts

Reply:
@zanssa - step one in the SOP / SLO updates - https://gitlab.cee.redhat.com/konflux/docs/sop/-/merge_requests/163

I'll start on the associated update to https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/stonesoup/pipeline-service/slo-sli-document.yml?ref_type=heads shortly

Reply:
step two is https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/112711 @amisstea @zanssa 

then I will update https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/stonesoup/pipeline-service/slo-sli-document.yml?ref_type=heads

thanks